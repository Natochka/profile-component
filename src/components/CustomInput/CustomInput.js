import styled from 'styled-components'
import { colors } from '../../styles/common-style'

const CustomInput = styled.input`
  font-size: 18px;
  color: ${colors.lightBlue};
  line-height: 22px;
  border: 0;
  border-bottom: 2px solid ${colors.lightGray};
  outline: none;
`

export default CustomInput
