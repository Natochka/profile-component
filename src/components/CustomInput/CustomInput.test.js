import React from 'react'
import { render } from 'react-testing-library'
import 'jest-dom/extend-expect'
import 'react-testing-library/cleanup-after-each'
import CustomInput from './CustomInput'

describe('CustomInput component test', () => {
  test('input should be rendered correctly', () => {
    const { container } = render(<CustomInput />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
