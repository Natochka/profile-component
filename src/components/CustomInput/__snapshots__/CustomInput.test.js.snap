// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomInput component test input should be rendered correctly 1`] = `
<input
  class="sc-bdVaJa coujGS"
/>
`;
